### DISCUSSION TITLE
Storing Multiple Repositories

### DISCUSSION THREAD

**Discussion by zachdoestein** [zeapo/Android-Password-Store#1886] [Q&A] [answered]
Hi, 

Is there a way to use/sync with multiple git repositories? I am trying to introduce this app (which is awesome!) to the team to share some passwords but I would like to keep my personal repository private.

Thanks!

**Comment by msfjarvis** [comment, marked as answer]
[There is not](https://msfjarvis.dev/aps/issue/260).

**Reply by zachdoestein** [reply]
Got it! Is this a feature expected be included in the future? or not considered at all? just curious.

**Reply by msfjarvis** [reply]
It's a feature that I do not expect to ever require, and thus do not foresee myself building.

**Reply by WhiteBlackGoose** [reply]
This feature would be really good. E. g. "personal passwords", "family passwords", etc.
Thanks for the app btw

**Comment by phaer** [comment]
As a kind of workaround: I use this app with an "external repository" in /sdcard/Gopass. It works, even with multiple sub repos, so i just used termux to clone each repo into /sdcard/Gopass/$repo and keep it in sync with a little shell alias calling "git pull --rebase && git push" in each sub-directory

**Comment by adam-devel** [comment]
Why would you need to use multiple repositories when you can just have multiple different folders with different ids in a single repository, no functionality is lost by representing a repository as a subfolder of a main repository

**Reply by WhiteBlackGoose** [reply]
Because I may want to sync with different servers?

**Reply by adam-devel** [reply]
That is fair

**Reply by ArchiFleKs** [reply]
Hi. Also someone may have multiple password store shared with different people / company. 

For example I use pass for my personal password manager and also as a company or shared password store for several project. 

Locally on my laptop it works greats with gopass and mounts for example. 

I'd love to be able to switch between password store on Android also.
